.github/workflows/ci.yml

```diff
@@ -26,4 +26,5 @@ jobs:
             -   run: composer install --no-progress
 #            -   run: composer codestyle
             -   run: composer phpstan
+                if:  matrix.php == '8.1'
```

Comment:
If we run PHPStan on every version, we'll use different versions of PHPStan detecting different sets of issues (due to bugs in older versions or whatnot).